waiter/src/waiter/client_tools.clj

```diff
@@ -191,17 +190,27 @@
             (log/warn (RuntimeException.))))
         (assoc request-headers "x-cid" new-cid)))))
 
-(defn- add-cookies
-  [waiter-url cookies ^CookieStore cs]
-  {:pre [(not (str/blank? waiter-url))]}
-  (doseq [cookie cookies]
-    (cookies/add-cookie! cs (str HTTP-SCHEME waiter-url) cookie)))
-
 (defn strip-trailing-slash
   "If s ends with /, returns s with the / stripped"
   [s]
   (cond-> s (str/ends-with? s "/") (subs 0 (- (count s) 1))))
 
+(defn parse-set-cookie-string
+  "Parse the value of a single Set-Cookie header."
+  [string-value]
```

Comment:
Should we just use a type hint `^String` and use the name `value`?

Reply:
Done